documentation/master-solicitor.asciidoc

```diff
@@ -1610,7 +1614,7 @@ Spring beans implementing this interface will be called at certain points in the
 [appendix]
 == Release Notes
 Changes in 1.15.0::
-
+* Add a new feature allowing the exclusion of paths and files of scanned artifacts from the Scancode information.
```

Comment:
Create an issue for Solicitor ("Enhancement") briefly describing the change and reference it here.

Reply:
https://github.com/devonfw/solicitor/issues/207